package.json

```diff
@@ -1,59 +1,61 @@
-{
-  "name": "wowerlay",
-  "description": "Wowerlay is a popover library for Vue 3 applications.",
-  "version": "0.7.0",

```

Comment:
Only version changed here, for no reason git is showing like whole file is changed.